src/main/java/com/zuniorteam/racingcar/core/RacingGame.java

```diff
@@ -0,0 +1,63 @@
+package com.zuniorteam.racingcar.core;
+
+import com.zuniorteam.racingcar.core.strategy.MovingStrategy;
+import com.zuniorteam.racingcar.core.vo.GameResult;
+import com.zuniorteam.racingcar.core.vo.StepResult;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+import static java.util.stream.Collectors.toList;
+
+public class RacingGame {
+
+    private final List<Car> cars;
+    private final int numberOfStep;
+
+    public RacingGame(int numberOfCars, int numberOfStep) {
+        this.cars = initCars(numberOfCars);
+        this.numberOfStep = numberOfStep;
+
+        validate();
+    }
+
+    private List<Car> initCars(int numberOfCars) {
+        return IntStream.rangeClosed(0, numberOfCars - 1)
```

Comment:
```suggestion
        return IntStream.range(0, numberOfCars)
```

로 작성하면 -1을 안해도 될 것 같은데 어떨까요?

Reply:
저도 같은 의견입니다

Reply:
오홍, 그런 메소드가있었군요, 감사합니다 👍